internal/kafka/internal/services/kafka_tls_certificate_management/kafka_tls_certificate_management_service.go

```diff
@@ -0,0 +1,223 @@
+package kafka_tls_certificate_management
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/config"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/shared"
+	"github.com/caddyserver/certmagic"
+	"github.com/libdns/route53"
+)
+
+// CertificateManagementOutput is the output indicating the certificates references
+type CertificateManagementOutput struct {
+	TLSCertRef string
+	TLSKeyRef  string
+}
+
+// GetCertificateRequest is the certificate request object.
+type GetCertificateRequest struct {
+	TLSCertRef string
+	TLSKeyRef  string
+}
+
+// areCertRefsDefined returns true if all certificate references are defined.
+// otherwise returns false
+func (req GetCertificateRequest) areCertRefsDefined() bool {
+	return !(shared.StringEmpty(req.TLSCertRef) || shared.StringEmpty(req.TLSKeyRef))
```

Comment:
This can be rewritten as return `!shared.StringEmpty(req.TLSCertRef) & !shared.StringEmpty(req.TLSKeyRef)`.
I am fine with both approaches

Reply:
I'll keep it to give @ziccardi a smile ;-)